Source/Engine/Core/Math/Quaternion.cs

```diff
@@ -1011,7 +1011,7 @@ public static Quaternion RotationLookAt(Vector3 forward, Vector3 up)
         /// <param name="forward">The forward direction. Direction to orient towards.</param>
         /// <param name="up">Up direction. Constrains y axis orientation to a plane this vector lies on. This rule might be broken if forward and up direction are nearly parallel.</param>
         /// <returns>The calculated quaternion.</returns>
-        public static Quaternion LookRotation(Vector3 forward, Vector3 up)
+        public static Quaternion LookRotation(Vector3 forward, Vector3 up = Vector3.Up)
```

Comment:
The default value must be a compile time constant, that's why it's not possible to set a custom value, C# limitation.
There is a workaround using nullable value but it's not very clean imo.

Reply:
fixed